VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -9236,19 +9426,19 @@ Function Request-PasswordRotationPolicy {
                     } elseif ($passwordRotation.expiry.expiryDate -le (Get-Date)) {
                         # If the password is expired, set the alert to red and the message to expired.
                         $message = 'Password is expired.'
-                        $alert = 'RED'   
+                        $alert = 'RED'
                     }
 
                     try {
                         $nextSchedule = [DateTime]::ParseExact($nextSchedule, 'yyyy-MM-ddTHH:mm:ss.fffZ', [System.Globalization.CultureInfo]::InvariantCulture)
-                    } catch {                                               
-                    } 
+                    } catch {
+                    }
                     $passwordRotationExpiryDate = $passwordRotation.expiry.expiryDate
                     try {
                         $passwordRotationExpiryDate = [DateTime]::ParseExact($passwordRotation.expiry.expiryDate, 'yyyy-MM-ddTHH:mm:ss.fffZ', [System.Globalization.CultureInfo]::InvariantCulture)
-                    } catch {                                             
-                    } 
-    
+                    } catch {
+                    }
```

Comment:
Catch block is empty, add a log message.

Reply:
Nice "catch"! I'll address this one shortly. 😀 

The other comments are resolved in the latest force push.

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/commit/c6c5af6dcb8e1f769c3432c4687b65dd40217af8.

Dropped in the use of `Debug-ExceptionWriter -object $_` in the two instances of an empty `] Catch [`.